include/envoy/stats/scope.h

```diff
@@ -52,6 +53,12 @@ class Scope {
    */
   virtual Counter& counterFromStatName(StatName name) PURE;
 
+  /**
+   * @param name The name of the stat, obtained from the SymbolTable.
+   * @return a counter within the scope's namespace.
```

Comment:
These still do tag extraction, but it might be a good idea to have it not do it? I don't imagine anyone would set tags and also expect to extract tags